README.md

```diff
@@ -39,6 +39,29 @@ You can also open an issue for a feature request.
 
 If you find an issue with this package and have a fix, please feel free to open a pull request following the [procedures](https://github.com/DataDog/datadog-agent/blob/master/docs/dev/contributing.md).
 
+## Testing
+
+To test a change to the Datadog Lambda Extension:
```

Comment:
We should specify that these instructions are just for testing in cloudrun. The process will be different when using aws lambda.